port_ocean/core/handlers/entity_processor/jq_entity_processor.py

```diff
@@ -230,6 +251,10 @@ async def _parse_items(
                         examples_to_send.append(result.raw_data)
                 else:
                     failed_entities.append(parsed_entity)
+        if len(entity_misconfigurations) > 0:
+            logger.info(
+                f"Entity mapping misconfigurations identified. misconfigured mapping: {entity_misconfigurations}"
```

Comment:
Explain what's going on here, Don't expect anyone seeing this will remember what they put in the mapping

Reply:
we've discussed this 🙏